# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/resource_cache.rs
# Commit: d6c9841c74cd
# Full Hash: d6c9841c74cdc5cdb84becbd304afcdebf13a3cf
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2019-04-17 21:47:29
# Regressor Bug: 1519106
# File Overlap Count: 1
# Description:
#   Bug 1519106 - Increase the maximum number of rasterized blob images per transaction. r=kats
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D16190
# ==============================================================================

diff -r b90fc7a65c3d -r d6c9841c74cd gfx/wr/webrender/src/resource_cache.rs
--- a/gfx/wr/webrender/src/resource_cache.rs	Wed Apr 17 17:43:51 2019 +0300
+++ b/gfx/wr/webrender/src/resource_cache.rs	Wed Apr 17 14:43:37 2019 +0000
@@ -1208,7 +1208,7 @@
                 // This code tries to keep things sane if Gecko sends
                 // nonsensical blob image requests.
                 // Constant here definitely needs to be tweaked.
-                const MAX_TILES_PER_REQUEST: i32 = 64;
+                const MAX_TILES_PER_REQUEST: i32 = 512;
                 // For truly nonsensical requests, we might run into overflow
                 // when computing width * height. Even if we don't, the loop
                 // below to reduce the number of tiles is linear and can take
